test/specs/productsCart.js

```diff
@@ -0,0 +1,59 @@
+import { expect, browser, $ } from '@wdio/globals';
+
+describe('Test 2 -Add product to the cart', () => {
+  it('Should log in to the "Swag Labs" page', async () => {
+    await browser.url(`/`);
+    await $('#user-name').setValue('standard_user');
+    await $('#password').setValue('secret_sauce');
+    await $('#login-button').click();
+
+    const actualUrl = await browser.getUrl();
+    const expectedUrl = 'https://www.saucedemo.com/inventory.html';
+
+    expect(actualUrl).toBe(expectedUrl);
+  });
+
+  it('Should add the first product to the cart', async () => {
+    const allAddToCartButtons = await $$(
+      'button.btn_primary.btn_small.btn_inventory'
+    );
+
+    if (allAddToCartButtons.length > 0) {
```

Comment:
I would better `expect(allAddToCartButtons).not.to.be.empty;` otherwise your test will pass without any assertions.
And test should contain an assertion in the end, otherwise we just dont have anything like Then step